src/site/content/en/blog/shadowdom/index.md

```diff
@@ -53,13 +53,11 @@ DOM changes, your styles and scripts might break in unpredictable
 ways.
 {% endAside %}
 
-Web Components is comprised of four
-parts:
+Web Components is comprised of three parts:
 
-1. [Templates](https://dvcs.w3.org/hg/webcomponents/raw-file/tip/spec/templates/index.html)
-1. [Shadow DOM](http://www.w3.org/TR/shadow-dom/)
-1. [Custom Elements](https://dvcs.w3.org/hg/webcomponents/raw-file/tip/spec/custom/index.html)
-1. [Packaging](https://dvcs.w3.org/hg/webcomponents/raw-file/tip/explainer/index.html#external-custom-elements-and-decorators)
```

Comment:
Removed this "packaging" thing. It's so old that I could not remember what it actually was 😄